howsmyssl.go

```diff
@@ -136,11 +136,22 @@ func main() {
 	}
 	oa := newOriginAllower(blockedOrigins, hostname, gclog, expvar.NewMap("origins"))
 
+	staticHandler := http.NotFoundHandler()
+	webHandler := func(w http.ResponseWriter, r *http.Request) {
```

Comment:
(maybe it should be webHandleFunc to clear up future confusion, but I leave that to your discretion)